server/data/licenceClientTypes.ts

```diff
@@ -21,14 +21,14 @@ export interface Case {
   booking_id: number
   stage: LicenceStage
   version: number
+  transition_date: Date
```

Comment:
added reportsService in typescript and needed `transition_date` as part of `Case` type so moved to `Case` to represent the db schema